src/gsy_e/models/strategy/market_agents/one_sided_engine.py

```diff
@@ -51,6 +51,20 @@ def __repr__(self):
             s=self
         )
 
+    def _update_offer_requirements_prices(self, offer):
```

Comment:
Atm this is not needed for the offers because the don't support 'energy' nor 'price' requirement. However this is also not harmful and will be needed in case the business logic changes.

Reply:
Thanks! Do we have some documentation/requirements about it where it would be possible to find this kind of information?

Reply:
It's not explicitly mentioned but in the following [board](https://miro.com/app/board/o9J_lAuQITw=/) you can see that the requirements are mostly for the bids.
The supported requirements are stored in a constant variable [here](https://github.com/gridsingularity/gsy-framework/blob/fd57017b8e7847dabb23168537ab7ffefbf8c7f0/gsy_framework/matching_algorithms/requirements_validators.py#L107)